tests/Doxyfile

```diff
@@ -656,6 +656,7 @@ WARN_LOGFILE           =
 INPUT                  = ../Bootloader/firmware/src \
                          ../common \
                          ../firmware/src \
+                         ../firmware/src/system_config/ethernet_sk2/system_settings.h \
```

Comment:
I realized last night I'm going to have to move this around a bit, since there are settings shared between the bootloader and the main app.

I think what I'll end up doing is creating a top level board_cfg directory, and then within that we'll have a template directory which people can copy when porting.